dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatMessage.cs

```diff
@@ -17,20 +17,27 @@ public abstract class ChatMessageBase
     /// <summary>
     /// Content of the message
     /// </summary>
-    public string Content { get; set; }
+    public string Content { get; set; } = string.Empty;
```

Comment:
Do we want to add any validation code to the setters to prevent Content/Role from being set to null?

Reply:
If a ChatMessage has just function calling the content can be null.

IMO this should be

```suggestion
    public string? Content { get; set; };
```

Reply:
I am reverting this change and i have a follow up task https://github.com/microsoft/semantic-kernel/issues/3522